scripts/eurostat/health_determinants/common/EuroStat.py

```diff
@@ -0,0 +1,359 @@
+                  "observationAbout: C:eurostat_population_{import_name}->geo\n"
+                  "observationDate: C:eurostat_population_{import_name}->time\n"
+                  "scalingFactor: 100\n"
+                  "value: C:eurostat_population_{import_name}->observation\n")
+
+
+class EuroStat:
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+
+    def __init__(self, input_files: list, csv_file_path: str,
+                 mcf_file_path: str, tmcf_file_path: str,
+                 import_name: str) -> None:
+        self._input_files = input_files
+        self._cleaned_csv_file_path = csv_file_path
+        self._mcf_file_path = mcf_file_path
+        self._tmcf_file_path = tmcf_file_path
+        self._import_name = import_name
+
+    def _parse_file(self, file_name: str, df: pd.DataFrame,
+                    import_name: dict) -> pd.DataFrame:
+        split_columns = df.columns.values.tolist()[0]
+        df = _split_column(df, split_columns)
+
+        split_columns = split_columns.replace('isced97', 'isced11')\
+                .replace('geo\time','geo').replace('geo\\time','geo')\
+                .replace('time\geo','time').replace('time\\geo','time')\
+                .replace('quantile', 'quant_inc')
+
+        df.rename(columns={
+            'geo\time': 'geo',
+            'geo\\time': 'geo',
+            'time\geo': 'time',
+            'time\\geo': 'time',
+            'isced97': 'isced11',
+            'quantile': 'quant_inc'
+        },
+                  inplace=True)
+
+        if import_name == "alcohol_consumption":
+            split_columns = split_columns.replace('frequenc',
+                                                  'frequenc_alcohol')
+            df.rename(columns={
+                'frequenc': 'frequenc_alcohol',
+            }, inplace=True)
+        elif import_name == "tobacco_consumption":
+            split_columns = split_columns.replace('frequenc',
+                                                  'frequenc_tobacco').replace(
+                                                      'quantile', 'quant_inc')
+            df.rename(columns={
+                'frequenc': 'frequenc_tobacco',
+                'quantile': 'quant_inc'
+            },
+                      inplace=True)
+        elif import_name == "bmi":
+            if 'quant_inc' in df.columns.values.tolist():
+                df = df[(~(df['quant_inc'] == 'UNK'))]
+
+        df = df[df['age'] == 'TOTAL']
+
+        for col in [
+                'sex', 'quant_inc', 'frequenc', 'frequenc_alcohol',
+                'frequenc_tobacco', 'isced11', 'deg_urb', 'bmi', 'duration',
+                'c_birth', 'citizen', 'smoking', 'duration', 'physact',
+                'lev_limit', 'levels'
+        ]:
+            if col in df.columns.values.tolist():
+                df = _replace_col_values(df, col)
+
+        df['SV'] = eval(file_to_sv_mapping[import_name][file_name])
```

Comment:
Can you generate a descriptive error message if file_name is not present in the dictionary  file_to_sv_mapping[import_name]?
```
if file_name not in file_to_sv_mapping[import_name]:
   print('Add a statement to generate SV in for {file_name} in common.sv-config.py')
```